test/bench/bench.cpp

```diff
@@ -27,12 +27,51 @@ using namespace benchmark;
 
 namespace
 {
-constexpr auto gas_limit = std::numeric_limits<int64_t>::max();
-auto vm = evmc::VM{};
+struct BenchmarkCase
+{
+    struct Input
+    {
+        std::string name;  ///< The input name, contains / prefix for convenience.
```

Comment:
Wouldn't it be simpler to add all required `/` at the point of call to `RegisterBenchmark` ?

Reply:
It was like that, but this allow us to add `""` input if there is none specified so we don't special case for "no inputs" at registration.

Reply:
It could be just a special case at benchmark name construction (unless input name is empty, add `/` + input name)

Now this comment is kind of not true for empty name.

Reply:
Fixed.